bin/i18n/hourofcode_crowdin.yml

```diff
@@ -2,25 +2,23 @@
 # Your crowdin's credentials
 #
 "project_identifier" : "hour-of-code"
-"base_path" : "../../i18n/locales/source/hourofcode"
+"base_path" : "../../i18n/locales"
 
 # API Credentials must be loaded from a separate identity file. See
 # https://support.crowdin.com/configuration-file/#split-project-configuration-and-api-credentials
 "api_key" : ""
 
-"preserve_hierarchy": true
-
 #
 # Files configuration
 #
 files: [
   {
-    "source": "/en.yml",
+    "source": "/source/hourofcode/en.yml",
     "translation": "/%language%/hourofcode/%original_file_name%",
     "update_option": "update_as_unapproved"
   }, {
-    "source": "/**/*.md",
-    "translation": "/%language%/hourofcode/**/%file_name%.md",
+    "source": "/source/hourofcode/**/*.md",
+    "translation": "%language%/hourofcode/**/%file_name%.md",
```

Comment:
is this intentional? Specifically, is it intentional that the translation field for `en.yml` has a leading slash and this one doesn't?

Reply:
oh that looks like a mistake. I'll fix that